clusterloader2/pkg/measurement/common/sleep.go

```diff
@@ -0,0 +1,54 @@
+package common
+
+import (
+	"fmt"
+	"time"
+
+	"k8s.io/klog"
+
+	"k8s.io/perf-tests/clusterloader2/pkg/measurement"
+	"k8s.io/perf-tests/clusterloader2/pkg/util"
+)
+
+const (
+	sleepName = "Sleep"
+)
+
+func init() {
+	if err := measurement.Register(sleepName, createSleepMeasurement); err != nil {
+		klog.Fatalf("Cannot register %s: %v", sleepName, err)
+	}
+}
+
+func createSleepMeasurement() measurement.Measurement {
+	return &sleepMeasurement{}
+}
+
+type sleepMeasurement struct{}
+
+func (s *sleepMeasurement) Execute(config *measurement.MeasurementConfig) ([]measurement.Summary, error) {
+	action, err := util.GetString(config.Params, "action")
```

Comment:
Do we even need to introduce "action" parameter for this measurement?

Reply:
+1, let's keep it simple